pandas/io/json/_normalize.py

```diff
@@ -305,8 +306,22 @@ def _recursive_extract(data, path, seen_meta, level=0):
                 # For repeating the metadata later
                 lengths.append(len(recs))
                 for val, key in zip(meta, meta_keys):
+                    # Extract the value of the key when the level
+                    # is at the meta path end
                     if level + 1 > len(val):
                         meta_val = seen_meta[key]
+                    # Extract the value of the key from seen_meta when
```

Comment:
can you put a blank line fore these comments (and below and above), basically easier to read if they are paragraph like.

Reply:
Definitely. Will update this tomorrow.